### DISCUSSION TITLE
Phone number provider to India

### DISCUSSION THREAD

**Discussion by archerabi** [supabase/supabase#18194] [Feature Requests]
Hello, the SMS providers that are currently available are prohibitively expensive for use in India. SMS login is by far the most prevelant login method in India owing to the extra protection provided by regulatory framework in terms of id verification. 

Please add an India specific SMS provider like https://msg91.com/in/pricing/sms which is 10x cheaper than the current providers.

**Comment by Srini-B** [comment]
There's a support for textlocal provider which is what I use. It's cheap and has INR pricing.

**Reply by archerabi** [reply]
Can you link to docs ? The official docs don't mention TextLocal.

**Reply by Srini-B** [reply]
Like you said, there are no docs. We just need to sign up and get the header and message body verified with DLT, or if you have it done before on another platform, you can import it to TextLocal. Then it's just adding your API key, how many SMS per hr and the message body. The dynamic variables used in message body on TextLocal can't be used on Supabase. For example, in TextLocal
`Your Login code is %%|otp^{"inputtype" : "text", "maxlength" : "6"}%% COH` 

becomes 

`Your Login code is {{ .Code }} COH`. There are different OTP categories, you need to choose yours accordingly, refer https://www.textlocal.in/service-promotional-transactional-sms/
